AmazonEmailService.pas

```diff
@@ -0,0 +1,166 @@
+unit AmazonEmailService;
+
+interface
+
+uses
+  Classes,
+  IPPeerAPI,
+  Data.Cloud.CloudAPI,
+  IPPeerClient;
+
+type
+  AwsRegions = (USEast, USWest, EUIreland);
```

Comment:
Sim @esasse. [Regions and Amazon SES](http://docs.aws.amazon.com/ses/latest/DeveloperGuide/regions.html)